bin/build-release

```diff
@@ -167,7 +167,9 @@ ln -snf buildroot-${BR_VER} buildroot
 # to have revision controlled information in that file.
 #./bin/mkcabundle > src/etc/ssl/certs/ca-certificates.crt
 
-( cd buildroot && QUILT_PATCHES="$PWD/../patches-buildroot" quilt push -a )
+# Note(frickler): We currently don't have any patches to apply, reenable
+# this once the situation changes
+#( cd buildroot && QUILT_PATCHES="$PWD/../patches-buildroot" quilt push -a )
```

Comment:
Could you change this to something like:

```patch
diff --git a/bin/build-release b/bin/build-release
index ea6a277..c019a4e 100755
--- a/bin/build-release
+++ b/bin/build-release
@@ -167,7 +167,12 @@ ln -snf buildroot-${BR_VER} buildroot
 # to have revision controlled information in that file.
 #./bin/mkcabundle > src/etc/ssl/certs/ca-certificates.crt
 
-( cd buildroot && QUILT_PATCHES="$PWD/../patches-buildroot" quilt push -a )
+( cd buildroot
+  rc=0
+  QUILT_PATCHES="$PWD/../patches-buildroot" quilt push -a || rc=$?
+  # quilt exits 0 on success, 2 on "nothing to do".
+  [ $rc -eq 0 ] || [ $rc -eq 2 ]
+)
 
 echo "$VER" > "src/etc/cirros/version"
 logevent "end unpack"
```

Reply:
quilt will also return rc=2 for some other errors, like no series file exists, so I don't like ignoring that return code completely. If you really think that commenting out is so bad, we could remove the empty series file completely and add a test "[ -f $QUILT_PATCHES/series ]" in front of the quilt invocation. We could also build a check for the file being non-empty but that'd be more complicated.

Reply:
I tested the resulting image, it works fine in terms of routes and MTUs. With dhcpcd also https://github.com/cirros-dev/cirros/pull/57 is no longer necessary (which is how I found that this cleanup is possible). I'll drop `src/etc/default/udhcpc` too, good catch.